docs/api-testing/composer.md

```diff
@@ -662,12 +609,16 @@ You can create specific assertions to verify performance metrics.
 #### Example
 The following is an example in **Code** view.
 
-  ```html
-  <assert-less expression="payload_response.metrics.latency" value="350" type="integer"/>
-
-  <assert-less expression="payload_response.metrics.fetch" value="350" type="integer"/>
-
-  <assert-less expression="payload_response.metrics.overall" value="550" type="integer"/>
+  ```yaml
```

Comment:
@aliciakrebs please pay attention to how you write the code because in yaml spaces and indentation are important

```yaml
- id: assert-less
   expression: payload_response.metrics.latency
   value: 350
- id: assert-less
   expression: payload_response.metrics.fetch
   value: 350
- id: assert-less
   expression: payload_response.metrics.overall
   value: 450
```